tools/SecurityTools/CredScanSuppressions.json

```diff
@@ -0,0 +1,1097 @@
+{
```

Comment:
Do you just copy the files from master branch? If yes, have you made any change after copying? I would suggest use first commit for exactly same files from master branch, then submit your changes using following commits, to reduce review time.

Reply:
first commit have the original file copied from master branch,
changes was made in commit:
65dadc63c676c125caa108483af300137478e045

Reply:
> Do you just copy the files from master branch? If yes, have you made any change after copying? I would suggest use first commit for exactly same files from master branch, then submit your changes using following commits, to reduce review time.

first commit have the original file copied from master branch,
changes was made in commit:
65dadc6